### PR TITLE
CLDR-17316 cldr-apps-webdriver add Dashboard test, etc

### PR DESCRIPTION
**Pull request by btangmu**
-New SurveyDriverDashboard.java implements TEST_DASHBOARD

-Add scripts/selenium-server-4.16.1.jar for convenience

-Configure pom.xml to use spotless formatter, same as cldr-apps

-Partially reformat (defer complete reformatting to subsequent PR)

-Partially update SurveyDriverXMLUploader.java per Survey Tool changes

-Turn off logging of 4444/status json, too noisy

-For the fast voting test (CLDR-17248), add scripts/cldrHarStats.mjs; generates summary (average post/get times) and html table (details)

-Comments

### CODE REVIEW

File: src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java [hunk 1]
```diff
@@ -61,10 +60,11 @@ public class SurveyDriver {
      * Enable/disable specific tests using these booleans
```
Comment (srl295) [inline src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java:60]: would it make sense for these to be in a properties or json file?

File: src/test/java/org/unicode/cldr/surveydriver/SurveyDriverDashboard.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    private final WebDriver driver;
+
+    public SurveyDriverDashboard(SurveyDriver s) {
+        this.s = s;
+        this.driver = s.driver;
+    }
+
+    private static final String[] locales = {
+        // Czech, German, Spanish, French, Hindi, Japanese, Russian, Chinese
+        "cs", "de", "es", "fr", "hi", "ja", "ru", "zh"
```
Comment (srl295) [inline src/test/java/org/unicode/cldr/surveydriver/SurveyDriverDashboard.java:19]: could also be in a config file

**Review by srl295 — APPROVED** [review]
looks great. only ideas for the future

File: src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java (hunk 1 above)
Comment (btangmu) [reply src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java:60]: yes, there's already a surveydriver.properties which so far only has the password in it

**Pull request merged**